src/client/pages/Offer/utils.ts

```diff
@@ -32,3 +35,18 @@ export const getAddress = (quotes: BundledQuote[]) => {
       })
     : ''
 }
+
+export const formatRegistrationNumber = (registrationNumber: string) => {
```

Comment:
should this also be named like `formatCarRegistrationNumber`?

Reply:
Yes, more explicit is good 👍  will fix!